### DISCUSSION TITLE
c1.1 version

### DISCUSSION THREAD

**Discussion by SoHailaNekHat** [Sharp-as-a-tech/chat-application#25] [Announcements]
## What's Changed
* Added a comment to AdminChatServlet class by @Yasnapopal in https://github.com/Sharp-as-a-tech/chat-application/pull/18

**Full Changelog**: https://github.com/Sharp-as-a-tech/chat-application/compare/c1.0...c1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Sharp-as-a-tech/chat-application/releases/tag/c1.1'>c1.1 version</a>.</em>

**Comment by Yasnapopal** [comment]
Great 👍

**Comment by zahidasafi** [comment]
that is good
